cinn/ir/ir_schedule.h

```diff
@@ -351,6 +351,21 @@ class IRSchedule {
   // TODO(sunli): Solve Index Simplify.
   void FlattenLoops(const std::vector<Expr>& loops, const bool force_flat = false);
 
+  /*!
+   * \brief Number of random sampling cycles for the loop to be split
```

Comment:
```suggestion
   * \brief Sample the factors to tile a specific loop perfectly
```
这句功能注释不太清晰，建议以表示`xx操作`动词开头